pkg/machine/qemu/machine.go

```diff
@@ -243,7 +243,11 @@ func (v *MachineVM) Init(opts machine.InitOptions) (bool, error) {
 		key string
 	)
 	sshDir := filepath.Join(homedir.Get(), ".ssh")
-	v.IdentityPath = filepath.Join(sshDir, v.Name)
+	targetIdentity, err := utils.FindTargetIdentityPath(sshDir, opts.Identity, "podman-machine", v.Port)
```

Comment:
i would imagine this change is also applicable to the hyperv and applehv implementations?

Reply:
applehv code should all be in main now .. it is not complete but booting it works.  See https://fedorapeople.org/groups/podman/testing/applehv/

Reply:
> applehv code should all be in main now
I noticed it. The parts, which were available before my change should be enough to implement the functionality for applehv (the change I added), but I can rebase to latest main if you insist. Will test applehv locally later this week.